frontend/class-twitter.php

```diff
@@ -161,6 +161,9 @@ protected function description() {
 		elseif ( WPSEO_Frontend::get_instance()->is_posts_page() ) {
 			$meta_desc = $this->single_description( get_option( 'page_for_posts' ) );
 		}
+		elseif ( is_category() || is_tax() || is_tag() ) {
```

Comment:
I don't quite understand what happens with $meta_desc. It gets set, but I don't get what for, as it's not returned in the return statement either.